### PR TITLE
pyfaf: Allow ignoring specific Fedora releases when pulling

### PR DESCRIPTION
**Pull request by mgrabovsky**

### CODE REVIEW

**Comment by mgrabovsky** [comment]
retest this please

File: src/pyfaf/opsys/fedora.py [hunk 1]
```diff
@@ -105,6 +106,7 @@ def __init__(self):
         self.pagure_url = None
         self.build_aging_days = None
         self.koji_url = None
+        self.ignored_releases = None
```
Comment (mkutlak) [inline src/pyfaf/opsys/fedora.py]:
``` java 
************* Module pyfaf.opsys.fedora                                                                                                                                                                                                        
pyfaf/opsys/fedora.py:369:23: E1133: Non-iterable value self.ignored_releases is used in an iterating context (not-an-iterable)
```

Comment (mgrabovsky) [reply src/pyfaf/opsys/fedora.py]: Meh, the variable should be a list once `load_config_to_self()` is finished below, but I don't blame pylint for missing it. Will change to default to `[]` here, as well.

**Comment by mkutlak** [comment]
LGTM. :+1:

**Pull request merged**